unitstatushandler/softwaremanager.go

```diff
@@ -213,6 +217,7 @@ func (manager *softwareManager) processRunStatus(status RunInstancesStatus) {
 }
 
 func (manager *softwareManager) processUpdateInstanceStatus(status []cloudprotocol.InstanceStatus) {
+	manager.InstanceStauses = status
```

Comment:
Update instances status is a diff from run status. It should not be handledd in software manager at all.